usaspending_api/api_contracts/contracts/v2/disaster/cfda/spending.md

```diff
@@ -102,6 +104,7 @@ Returns spending details of CFDA receiving supplemental funding budgetary resour
 + `obligation` (required, number, nullable)
 + `outlay` (required, number, nullable)
 + `total_budgetary_resources` (required, number, nullable)
++ `sam_website` (required, string, nullable)
```

Comment:
Would prefer this to be generic, like `website_link`, `website`, `webpage`, `resource_link` or simply `link`. Thoughts?

Reply:
I was using the same name as in the [awards contract CFDA object](https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/api_contracts/contracts/v2/awards/award_id.md#cfdainfo-object) but am good with making it more generic here!

Reply:
https://media.giphy.com/media/l2RaPhzQeY4ODghNK/source.gif